acceptance_tests/features/telephone_capture.feature

```diff
@@ -0,0 +1,8 @@
+
+Feature: Telephone capture
+
+  Scenario: A case can be completed using telephone capture
+    Given sample file "sample_1_limited_address_fields.csv" is loaded successfully
+    When there is a request for telephone capture of a case
```

Comment:
I'd prefer this to be one step of something like:
A request for telephone capture of the 1st case returns a UAC and QID of type "01"

I'm not sure of the benefit of having it split over 2 steps, and putting more 'stuff' in the context magic